### ISSUE TITLE
jsonlines

### ISSUE THREAD

**Issue by glensc** [ndjson/ndjson-spec#35]
I was suprised there's another json lines "standard":

- http://ndjson.org/
- https://jsonlines.org/

don't know which originated from what (or developed independently), but would be nice to have a comparison with jsonlines.

ps: ndjson.org website is derived from jsonlines.org website:

![image](https://user-images.githubusercontent.com/199095/109829971-b28be680-7c46-11eb-8ad0-813220f2f9af.png)

**Comment by glensc** [comment]
found similar issue in the jsonlines repo:
- https://github.com/wardi/jsonlines/issues/22

**Comment by pekkaklarck** [comment]
:+1: for just one standard and one web site. Based on a quick look there aren't any real differences except to the extension (.`jsonl` vs `.ndjson`). Having a common extension would make it more likely that editors and IDEs support this format without extra configuration.

**Comment by jsejcksn** [comment]
I know this issue isn't the first [duplicate](https://github.com/ndjson/ndjson-spec/issues/8), and I understand that, historically, there were spec differences due to potential ambiguity (UTF-8 encoding, required JSON data on every line, etc.), but it seems as though they are now aligned. At this point in time, are there any remaining spec differences?

And are there any other issues which are preventing convergence (e.g. copyright credit, etc.)? I think the community will greatly benefit from a single, unified standard with an RFC, registered IANA media type, etc. The involved parties appear to be reasonable and [responsive](https://github.com/wardi/jsonlines/issues/22#issuecomment-1176738602). Can we make this happen?

---

Related observation: If repository issue activity is any metric for discoverability, then the _JSON Lines_ name has an advantage.

**Comment by maximveksler** [comment]
Yay for merge of formats.

Also - Please maintain the `jsonl` extension :+1:

**Comment by finnp** [comment]
I'm the owner of the ndjson domain and have contributed to the original spec. I'm no longer working with my employer at the time who was interested in the standardisation of NDJSON, so indeed I'm not very motivated to continue the work here and submit a RFC.

That being said. I don't really care about the name and I don't think it's very useful to have two different websites and descriptions for this. Especially since I don't see any differences between the formats.

I'm open to the idea to deprecate the ndjson repository and redirect the website to jsonlines.org (since some places link it). 

Maybe it would be nice to keep the spec document and update it for the new name? Maybe still mentioning the former name.

**Comment by sp4ce** [comment]
Hi, I am trying to build the RFC to standardize jsonlines (https://github.com/wardi/jsonlines/issues/19), and in an effort to avoid having both ndjson and jsonl around, that would be great if you could deprecate ndjson and we would mention the name in the RFC as well.

**Comment by markusschaber** [comment]
I think there's actually a difference between NDJSON and JsonLines: NDJSON optionally allows parsers to skip empty lines. I don't know how relevant this is in real world, but it may be useful to add this to jsonlines before deprecating NDJSON.

**Comment by remram44** [comment]
> NDJSON optionally allows parsers to skip empty lines

This "optionally" is scary. It would be best if the spec was clear on what is and isn't a valid file.

**Comment by KaKi87** [comment]
Hi,

Why is the website displaying chinese stuff ?

**Comment by jsejcksn** [comment]
> Hi,
> 
> Why is the website displaying chinese stuff ?

@KaKi87 Please open a new issue for a new topic.

**Comment by KaKi87** [comment]
I'm asking here because @finnp said in this very thread :

> I'm open to the idea to deprecate the ndjson repository and redirect the website to jsonlines.org

**Comment by smurp** [comment]
TL;DR JSONL has had more uptake than NDJSON (but NDJSON has a mimetype!  This is why we can't have nice things)

FYI  here is a view of the relative impact of JSONL vs NDJSON

https://trends.google.com/trends/explore?date=all&q=jsonl,ndjson&hl=en-GB

**Comment by sp4ce** [comment]
Hi @smurp 
> but NDJSON has a mimetype!

If you are refering to `application/x-ndjson`, this is not a standard type, so I am not sure what you mean. I have been trying to have `application/jsonl` being adopted more widely, so you feel free to start using it.
